### DISCUSSION TITLE
Outlook Links Won't Open

### DISCUSSION THREAD

**Discussion by tykisson** [memstechtips/UnattendedWinstall#138] [General]
I have been using the UnattendedWinInstall for several computers and have an issue where users were unable to click and open links in Outlook. We do not block this yet in our enviroment so it had to be something with the removal of IE during the install.

I did confirm a resolution by added the following keys in the Registry:

1.) Going to registry (regedit) **make sure to backup your registry before making any changes**

2.) Navigating to Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Classes\htmlfile\shell

3.) Under shell, create new key called "open"

4.) Under open, create another new key called "command"

**Comment by memstechtips** [comment]
Hey there and thanks for reporting on this.
